pkg/bmc/bmc.go

```diff
@@ -44,14 +56,17 @@ type BMC struct {
 	host        string
 	redfishUser User
 	sshUser     User
+	sshPort     int
 	systemIndex int
 
 	timeOuts TimeOuts
+
+	sshSessionForSerialConsole *ssh.Session
 }
 
 // New returns a new BMC struct. The default system index to be used in redfish requests is 0.
 // Use SetSystemIndex to modify it.
-func New(host string, redfishUser, sshUser User, timeOuts TimeOuts) (*BMC, error) {
+func New(host string, redfishUser, sshUser User, sshPort int, timeOuts TimeOuts) (*BMC, error) {
```

Comment:
May be ssh port has to be uint instead of int?

Reply:
Done in https://github.com/openshift-kni/eco-goinfra/pull/419/commits/74bdadb9f818f05f583fe09fd8569c0f403eb56e